### PR TITLE
Move files

### PR DESCRIPTION
**Pull request by jacob314**
This is needed so that package constraints in package:devtools don't impact who can run devtools with pub global activate.

We should be able to safely develop the devtools_app on bleeding edge Dart and Flutter without impacting users on much older versions. package:devtools now only includes the code to serve devtools and a binary snapshot of the devtools app compiled to dart2js.
This pull request has two commits. One updates all our docs and scripts to use the new structure. The other moves everything.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 			"label": "Serve DDC",
 			"type": "shell",
 			"options": {
-				"cwd": "${workspaceFolder}/packages/devtools",
+				"cwd": "${workspaceFolder}/packages_app/devtools",
 			},
 			"command": "pub",
```
Comment (kenzieschmoll) [inline .vscode/tasks.json]: Should this be devtools_app?

File: fixtures/flutter_app/android/local.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+sdk.dir=/Users/jacobr/Library/Android/sdk
```
Comment (kenzieschmoll) [inline fixtures/flutter_app/android/local.properties]: Do we need this file?

File: fixtures/flutter_app/build/frontend_server.d [hunk 3]
```diff
@@ -0,0 +1 @@
+frontend_server.d: /Users/jacobr/flutter/bin/cache/artifacts/engine/darwin-x64/frontend_server.dart.snapshot
```
Comment (kenzieschmoll) [inline fixtures/flutter_app/build/frontend_server.d]: can we remove this file? anything with your username in it seems suspicious 🕵️‍♀️

File: fixtures/flutter_app/ios/Flutter/Generated.xcconfig [hunk 4]
```diff
@@ -0,0 +1,9 @@
+// This is a generated file; do not edit or check into version control.
+FLUTTER_ROOT=/Users/jacobr/git/flutter/2/flutter
```
Comment (kenzieschmoll) [inline fixtures/flutter_app/ios/Flutter/Generated.xcconfig]: remove this file?

File: packages/devtools_app/pubspec.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+name: devtools_app
+description: Main package implementing web-based performance tooling for Dart and Flutter.
+
+# Note: when updating this version, please update the corresponding entry in
+# lib/devtools.dart and publish a new version of package:devtools as well.
```
Comment (kenzieschmoll) [inline packages/devtools_app/pubspec.yaml]: update comment to include both files that need to be updated.

File: packages/devtools_app/LICENSE [hunk 6]
```diff
@@ -0,0 +1,27 @@
+Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/LICENSE]: nit: 2019?

File: .vscode/tasks.json (hunk 1 above)
Comment (jacob314) [reply .vscode/tasks.json]: yes. package:devtools is now just a placeholder for the deployed artifacts. When debugging, you just need package:devtools_app.

File: fixtures/flutter_app/android/local.properties (hunk 2 above)
Comment (jacob314) [reply fixtures/flutter_app/android/local.properties]: good catch. this directory was actually inserted. Not sure why my client had those spurious files. The fixtures are now all under the separate devtools_testing package.

File: fixtures/flutter_app/ios/Flutter/Generated.xcconfig (hunk 4 above)
Comment (jacob314) [reply fixtures/flutter_app/ios/Flutter/Generated.xcconfig]: done

File: packages/devtools_app/LICENSE (hunk 6 above)
Comment (jacob314) [reply packages/devtools_app/LICENSE]: This package is not materially change from what was in package:devtools so keeping the year the same is right.

File: fixtures/flutter_app/build/frontend_server.d (hunk 3 above)
Comment (jacob314) [reply fixtures/flutter_app/build/frontend_server.d]: removed this whole directory.

File: packages/devtools_app/pubspec.yaml (hunk 5 above)
Comment (jacob314) [reply packages/devtools_app/pubspec.yaml]: done.

File: .vscode/tasks.json (hunk 1 above)
Comment (kenzieschmoll) [reply .vscode/tasks.json]: I mean should this say `packages/devtools_app` instead of `packages_app/devtools`

Comment (jacob314) [reply .vscode/tasks.json]: done... wow.... `packages_app`.

File: packages/devtools_app/pubspec.yaml [hunk 7]
```diff
@@ -0,0 +1,63 @@
+name: devtools_app
+description: Main package implementing web-based performance tooling for Dart and Flutter.
+
+# Note: when updating this version, please update the corresponding entry in
+# lib/devtools.dart and ../devtools/pubspec.yaml as well to the same version.
+# When publishing new versions of this package be sure to publish a new version
+# of package:devtools as well. package:devtools contains a compiled snapshot of
+# this package.
```
Comment (kenzieschmoll) [inline packages/devtools_app/pubspec.yaml]: maybe add a note about which package needs to be published first if order matters?

Comment (jacob314) [reply packages/devtools_app/pubspec.yaml]: doesn't matter as package_devtools doesn't depend on devtools_app

**Pull request merged**